<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyInfo</class>
 <widget class="QWidget" name="modifyInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SearchBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddrLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="IDLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>身份证：</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddrLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>住址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>0</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;会员信息修改&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回上一层</string>
   </property>
  </widget>
  <widget class="QLabel" name="PhoneLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="BGRLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/backimage.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SIDLabel">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输要修改会员的身份证：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SIDLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <zorder>BGRLabel</zorder>
  <zorder>modifyBtn</zorder>
  <zorder>NameLineEdit</zorder>
  <zorder>IDLineEdit</zorder>
  <zorder>AddrLineEdit</zorder>
  <zorder>PhoneLineEdit</zorder>
  <zorder>NameLabel</zorder>
  <zorder>AddrLabel</zorder>
  <zorder>IDLabel</zorder>
  <zorder>PhoneLabel</zorder>
  <zorder>SearchBtn</zorder>
  <zorder>TitleLabel</zorder>
  <zorder>ReturnBtn</zorder>
  <zorder>SIDLabel</zorder>
  <zorder>SIDLineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
